frappe/commands/utils.py

```diff
@@ -923,6 +923,12 @@ def run_ui_tests(
 @click.command("serve")
 @click.option("--port", default=8000)
 @click.option("--profile", is_flag=True, default=False)
+@click.option(
+	"--proxy",
+	is_flag=True,
+	default=False,
+	help="The development server may be run behind a proxy, e.g. ngrok / localtunnel",
+)
```

Comment:
Maybe add some validation to alert the user that these cannot be used in combination?

Reply:
I don't see why they couldn't. Do you have more info on the incompatibility?

I'm by no means an authority in the field and could have easily missed something, but this looks actually fine to me:
```python
application = ProfilerMiddleware(application, sort_by=("cumtime", "calls"))application = ProxyFix(application, x_for=1, x_proto=1, x_host=1, x_port=1, x_prefix=1)
```

Reply:
Ah, right, I overlooked that they use the same application.